test_util/cluster_api.py

```diff
@@ -338,6 +340,11 @@ def get_test_app_in_docker(self, ip_per_container):
         return app, test_uuid
 
     def deploy_test_app_and_check(self, app, test_uuid):
+        marathon_user = self.mesos_user
+        # if using docker containerizer, reported user will be docker user
```

Comment:
Could you expand on this comment a little? I think it could be helpful to explain that the testing service provided by the test app checks the username within the task's container; when using the docker containerizer, this user is dependent on the image being used.